<ui version="4.0" >
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Search Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="1" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="searchPushButton" >
           <property name="text" >
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settingsPushButton" >
           <property name="text" >
            <string>Settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closePushButton" >
           <property name="text" >
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item rowspan="2" row="0" column="0" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Search for:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="searchLineEdit" />
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox" >
           <property name="title" >
            <string>Options</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QCheckBox" name="csCheckBox" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Case sensitive search</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
              <property name="tristate" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="sfCheckBox" >
              <property name="text" >
               <string>Search in filenames </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="sdiCheckBox" >
              <property name="text" >
               <string>Search in directory names</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="sdeCheckBox" >
              <property name="text" >
               <string>Search in descriptions</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="resultsTreeWidget" />
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>searchPushButton</tabstop>
  <tabstop>settingsPushButton</tabstop>
  <tabstop>closePushButton</tabstop>
  <tabstop>csCheckBox</tabstop>
  <tabstop>sfCheckBox</tabstop>
  <tabstop>sdiCheckBox</tabstop>
  <tabstop>sdeCheckBox</tabstop>
  <tabstop>resultsTreeWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>670</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel" >
     <x>359</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
